### DISCUSSION TITLE
Keyboard shortcut cmd + p does not work in Safari

### DISCUSSION THREAD

**Discussion by cbrnr** [community/community#60151] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

I set my search palette shortcut to cmd + p, which works in Chrome. However, Safari shows the Print dialog instead. Interestingly, both Chrome and Safari map cmd + p to the Print dialog, but for some reason, I still get the GitHub search palette with that shortcut in Chrome, but not in Safari. Is there a way to make cmd + p work in Safari?

**Comment by GhostOf0days** [comment, marked as answer]
Hi if you changed it [here](https://support.apple.com/guide/mac-help/create-keyboard-shortcuts-for-apps-mchlp2271/mac) and it still didn't work, it's an issue with Apple reserving that shortcut and not enabling you to change it.

**Reply by cbrnr** [reply]
I didn't create that shortcut myself, Safari (and Chrome) use cmd + p for printing by default.

**Reply by GhostOf0days** [reply]
I understand. Regardless, you should try overriding that shortcut or see if you can disable it. If it's reserved by Apple (as in unchangeable), you can't use custom shortcuts for that key combo like the one you set for your search palette.

**Reply by cbrnr** [reply]
OK, thanks. So are you saying that even though Chrome uses the same shortcut, they have somehow disabled it when I'm on GitHub? Because the menu entry still shows the shortcut for Print, but it opens the palette.

**Reply by GhostOf0days** [reply]
Apple disables a lot of custom stuff. They used to not allow .webp images while a lot of other browsers did. They changed that in Safari version 14.

**Reply by cbrnr** [reply]
Thanks! I'll try Safari 14 then and hope they'll allow changing that shortcut.

**Reply by GhostOf0days** [reply]
Safari version 14 was in reference to them changing the ability to use .webp images on the website. If they disabled the key combo on the latest versioon of Safari, it won't work on Safari version 14.

**Reply by cbrnr** [reply]
Sorry for the confusion. I meant to say I'll try the next Safari release once it's out (presumably with the next OS update).
